### DISCUSSION TITLE
Description doesn't appear to load on Firefox

### DISCUSSION THREAD

**Discussion by brendon-wong** [community/community#15237] [Projects and Issues]
I just see a spinning circle that never goes away on Firefox, whereas the description loads instantly on Chrome

<img width="1264" alt="Screen Shot 2022-04-19 at 4 57 42 PM" src="https://user-images.githubusercontent.com/6268499/164120131-3d00737a-7696-4127-ba08-b6ce212194bc.png">

**Comment by xmo-odoo** [comment]
It seems to behave strangely on newly created projects: I created a project to test things, created an entry, opened it, it started spinning its wheels same as you then the entire browser crashed.

After the browser restarted, the entry opened fine and instantaneously.
